packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -557,40 +557,11 @@ class Viewport implements IViewport {
       imageArea = { areaX: 1, areaY: 1 },
       imageFocalPoint = { focalX: 0.5, focalY: 0.5 },
     } = displayArea;
-    // setPan functions requires offsetviewer to be initialized, which does not seem to be the case at constructor time
-    // vtkWindowRenderer.displayToWorld seems to always return NaN until image passes first render?
-    const imageData = this.getDefaultImageData();
-    if (imageData) {
-      const dimensions = imageData.getDimensions();
-      const idx = [
-        Math.floor(dimensions[0] * imageFocalPoint.focalX),
-        Math.floor(dimensions[1] * imageFocalPoint.focalY),
-        Math.floor(dimensions[2] * 0.5), // always use middle of imageData, even for volume viewport which clips bounds to center
-      ];
-      const newFocalPoint = <Point3>[0, 0, 0];
-      imageData.indexToWorld(idx, newFocalPoint);
-      const previousCamera = this.getCamera();
-      const { focalPoint, position } = previousCamera;
-      const delta = vec3.subtract(vec3.create(), newFocalPoint, focalPoint);
-      const newFocal = vec3.subtract(
-        vec3.create(),
-        focalPoint,
-        vec3.multiply(vec3.create(), delta, <Point3>[2, 2, 2])
-      );
-      const newPosition = vec3.subtract(
-        vec3.create(),
-        position,
-        vec3.multiply(vec3.create(), delta, <Point3>[2, 2, 2])
-      );
-      this.setCamera(
-        {
-          ...previousCamera,
-          focalPoint: newFocal as Point3,
-          position: newPosition as Point3,
-        },
-        storeAsInitialCamera
-      );
-    }
+
+    this.setPan(
+      [imageFocalPoint.focalX, imageFocalPoint.focalY],
```

Comment:
@diegohennrich, this will require a diff calculation. We need to get the current camera focalX and focalY and subtract it from the imageFocalPoint X and Y. I believe the setPan requires a delta input, not an absolute input value.